api/applications/models/application_patchrequest.go

```diff
@@ -34,4 +34,9 @@ type ApplicationPatchRequest struct {
 	//
 	// required: false
 	ConfigBranch *string `json:"configBranch,omitempty"`
+
+	// AcknowledgeWarnings acknowledge all warnings
+	//
+	// required: false
+	AcknowledgeWarnings bool `json:"acknowledgeWarnings,omitempty"`
```

Comment:
Refactor ApplicationPatchRequest: add ApplicationPatch struct containing fields to be patched, and include ApplicationPatch as field Patch in ApplicationPatchRequest

Reply:
Fixed